tests/unit-core/test-mem-stats.c

```diff
@@ -16,7 +16,6 @@
 #include "jerryscript.h"
 #include "test-common.h"
 
-#ifdef JMEM_STATS
 
 const char *test_source = (
```

Comment:
I am not sure about this to keep it here, If it is only used within this file. Now it works because it is not static, but I think it should be declare as "static" here (I suppose it is not used elsewhere). Then the GCC will warn you, if it is not used (JERRY_FEATURE_MEM_STATS is not enabled). I suggest you to move this variable inside the function, right before it is used.